### PR TITLE
Fix build error on IOS

### PR DESCRIPTION
**Pull request by moneromooo-monero**

### CODE REVIEW

File: src/wallet/CMakeLists.txt [hunk 1]
```diff
@@ -121,7 +121,6 @@ if (BUILD_GUI_DEPS)
             wallet
             rpc_base
             multisig
-            blockchain_db
```
Comment (xiphon) [inline src/wallet/CMakeLists.txt]: `cryptonote_core` depends on `blockchain_db`, so we have to link `blockchain_db` as well

Comment (moneromooo-monero) [reply src/wallet/CMakeLists.txt]: Does the GUI wallet depend on cryptonote_core ? I don't see explicit deps.

Comment (xiphon) [reply src/wallet/CMakeLists.txt]: GUI depends on `wallet_api`, `wallet_api` depends on `cryptonote_core`

Comment (moneromooo-monero) [reply src/wallet/CMakeLists.txt]: OK, then I'll leave it to someone who builds the GUI.

**Pull request closed without merging**